functional_tests.go

```diff
@@ -701,49 +848,57 @@ func testGetObjectClosedTwice() {
 
 	// Save the data
 	objectName := randString(60, rand.NewSource(time.Now().UnixNano()), "")
+	args["objectName"] = objectName
+
 	n, err := c.PutObject(bucketName, objectName, reader, "binary/octet-stream")
 	if err != nil {
-		logger().Fatal("Error:", err, bucketName, objectName)
+		failureLog(function, args, startTime, "", "PutObject failed", err).Fatal()
 	}
 
 	if n != int64(thirtyThreeKiB) {
-		logger().Fatalf("Error: number of bytes does not match, want %v, got %v\n", thirtyThreeKiB, n)
+		failureLog(function, args, startTime, "", "PutObject response doesn't match sent bytes, expected "+string(int64(thirtyThreeKiB))+" got "+string(n), err).Fatal()
 	}
 
 	// Read the data back
 	r, err := c.GetObject(bucketName, objectName)
 	if err != nil {
-		logger().Fatal("Error:", err, bucketName, objectName)
+		failureLog(function, args, startTime, "", "GetObject failed", err).Fatal()
 	}
 
 	st, err := r.Stat()
 	if err != nil {
-		logger().Fatal("Error:", err, bucketName, objectName)
+		failureLog(function, args, startTime, "", "Stat failed", err).Fatal()
 	}
 	if st.Size != int64(thirtyThreeKiB) {
-		logger().Fatalf("Error: number of bytes in stat does not match, want %v, got %v\n",
-			thirtyThreeKiB, st.Size)
+		failureLog(function, args, startTime, "", "Number of bytes in stat does not match, expected "+string(int64(thirtyThreeKiB))+" got "+string(st.Size), err).Fatal()
 	}
 	if err := r.Close(); err != nil {
-		logger().Fatal("Error:", err)
+		failureLog(function, args, startTime, "", "Object Close failed", err).Fatal()
 	}
 	if err := r.Close(); err == nil {
-		logger().Fatal("Error: object is already closed, should return error")
+		failureLog(function, args, startTime, "", "Already closed object. No error returned", err).Fatal()
```

Comment:
`err` is nil. No point in logging it in.
`failureLog(function, args, startTime, "", "Supposed to error out for the already closed object"~~, err~~).Fatal()`